src/events/ready.ts

```diff
@@ -49,7 +49,7 @@ export class ReadyEvent extends ClientEvent<'ready'> {
       return acc;
     }, []);
     for (const [guild, guildChannels] of guildChannelsToSetup) {
-      setupRandomVoiceChannelNamesCron(guild, guildChannels);
+      await setupRandomVoiceChannelNamesCron(guild, guildChannels);
```

Comment:
Might be able to swap this with a `Promise.all`

Reply:
Swapping this for a map and Promise.all would basically be the same for performance, no?